### PR TITLE
Core: StringProperty: Keep linebreaks in tooltips

### PR DESCRIPTION
**Pull request by r-englund**
Before fix:
image
After fix:
image

### CODE REVIEW

**Review by petersteneteg — DISMISSED** [review]
I would have put the string in a &lt;pre> element instead, but I guess this works to

**Comment by r-englund** [comment]
&lt;pre> changes the font to a monospace font, in the example above where there is code in the string property that might make sense, but in other cases it feels strange, for example
image

**Comment by r-englund** [comment]
Hrm, ´pre´ might still be preferable, since it keeps extra whitespaces etc. (Code indentation for the if-statement is missing in the example above)

**Comment by r-englund** [comment]
image

**Comment by martinfalk** [comment]
That's a long tooltip ;-) Might cause some problems on smaller screen resolutions in general.

File: src/core/properties/stringproperty.cpp [hunk 1]
```diff
@@ -51,7 +51,9 @@ Document StringProperty::getDescription() const {
     using P = Document::PathComponent;
     Document doc = TemplateProperty<std::string>::getDescription();
     auto b = doc.get({P("html"), P("body")});
-    b.append("p", value_.value);
+    auto val = value_.value;
+    replaceInString(val, "\n", "<br \>\n");
```
Comment (martinfalk) [inline src/core/properties/stringproperty.cpp]: should read `<br />` !

**Review by martinfalk — CHANGES_REQUESTED** [review]

**Review by martinfalk — DISMISSED** [review]

**Pull request closed without merging**